### PR TITLE
DOC: add banner for NF survey

### PR DESCRIPTION
**Pull request by tacaswell**
This will need to be reverted at the end of the month.

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -42,11 +42,20 @@
...
+by <a href="https://numfocus.org/">NumFOCUS</a> to understand diversity,
+inclusion and barriers to participation within NumFOCUS-sponsored projects and
+the wider open source community.
+
+
+<a href="https://numfocus.typeform.com/to/W6Bax8eq">The survey</a> will take
+15-20 mn to complete. We’d appreciate your contribution by May 31, 2021.
+
+
+<a href="https://numfocus.typeform.com/to/W6Bax8eq"> Click here to participate into the survey</a>
```
Comment (jklymak) [inline site/index.html:56]:
```suggestion
<a href="https://numfocus.typeform.com/to/W6Bax8eq"> Click here to participate in the survey</a>
```

File: site/index.html [hunk 2]
```diff
@@ -42,11 +42,20 @@
   </head><body>
 
 
-<!--
-<div id="annc-banner">
+<div id="annc-banner" style="padding:10px; background-color:#eff9ff; background-image:none;">
+Matplotlib is participating in a research project being conducted
+by <a href="https://numfocus.org/">NumFOCUS</a> to understand diversity,
+inclusion and barriers to participation within NumFOCUS-sponsored projects and
```
Comment (jklymak) [inline site/index.html:48]:
```suggestion
inclusion and barriers to participation in NumFOCUS-sponsored projects and
```

File: site/index.html [hunk 3]
```diff
@@ -42,11 +42,20 @@
...
+the wider open source community.
+
+
+<a href="https://numfocus.typeform.com/to/W6Bax8eq">The survey</a> will take
+15-20 mn to complete. We’d appreciate your contribution by May 31, 2021.
```
Comment (jklymak) [inline site/index.html:53]:
```suggestion
15-20 mn to complete. We would appreciate your contribution by May 31, 2021.
```

**Review by jklymak — APPROVED** [review]

**Review by timhoffm — APPROVED** [review]

**Comment by tacaswell** [comment]
Hmm, may have to adjust who has write permissions to this...

**Pull request merged**